armory/utils/external_repo.py

```diff
@@ -28,13 +28,22 @@ def download_and_extract_repo(
 
     os.makedirs(external_repo_dir, exist_ok=True)
     headers = {}
-    repo_name = external_repo_name.split("/")[-1]
+
+    if "@" in external_repo_name:
+        split_repo = external_repo_name.split("@")
+        org_repo_name = split_repo[0]
+        repo_name = org_repo_name.split("/")[-1]
+        branch = split_repo[1]
+    else:
+        branch = "master"
+        org_repo_name = external_repo_name
+        repo_name = external_repo_name.split("/")[-1]
```

Comment:
Nit. Proposed change to remove redundant code:
```python
    if "@" in external_repo_name:                                                        
        org_repo_name, branch = external_repo_name.split("@")                            
    else:                                                                                
        org_repo_name = external_repo_name                                               
        branch = "master"                                                                
    repo_name = external_repo_name.split("/")[-1]
```

Reply:
Thanks, much better